pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -723,10 +723,7 @@ self: super: builtins.intersectAttrs super {
     '';
   }) super.haskell-language-server;
 
-  # NOTE: this patch updates the hevm code to work with the latest packages that broke the build
-  # it's temporary until hevm version 0.50.0 is released - https://github.com/ethereum/hevm/milestone/1
-  # tests depend on a specific version of solc
-  hevm = dontCheck (appendPatch ./patches/hevm-update-deps.patch super.hevm);
+  hevm = dontCheck super.hevm;
```

Comment:
Could you leave a comment about why tests are disabled?

Thanks for removing the patch!

Reply:
done